### ISSUE TITLE
Create sample using gRPC for .NET Core

### ISSUE THREAD

**Issue by JayBeavers** [google-research/mozolm#94]
See this:  https://github.com/grpc/grpc-dotnet

Would be willing to work on an equivalent example to the SimpleJavaSample, but written in .NET Core for easier usability on the Windows platform.

Would a PR here be welcomed?

**Comment by agutkin** [comment]
Absolutely, Jay. PRs are welcome even though we haven't set up the scaffolding that allows us to ingest external changes to our github repo automatically into our own corporate version control -- at the moment I am doing these merges manually, which isn't a huge deal given that this is example code (i.e. the current work flow for me is: merge on github, merge by hand internally, re-sync to github).

I am not sure how well `grpc-dotnet` integrates with Bazel (it may be the early days), but the example doesn't have to be Bazel buildable.

I'll assign this to you.
